### DISCUSSION TITLE
What does `"onSave": "test-only"` do?

### DISCUSSION THREAD

**Discussion by JordanLongstaff** [orta/vscode-jest#803] [Q&A]
I've seen this option in the README, but I don't see it anywhere in the code. I thought that its intended purpose was to watch for a test file to be saved and then run only the tests that appeared to be modified, but I'm making edits to a test file that are completely pointless and purposely situated outside of any actual test in the file, and then Jest runs all the tests in the file.

**Comment by connectdotz** [comment]
if you turned off the watch mode and configured onSave then it should trigger jest run when you save a file jest considered as test file, i.e. it should be within the list returned by `jest --iistTests`. if it's not doing that, then you could create an issue with a sample repo so we can dig deeper

**Reply by JordanLongstaff** [reply]
OK...and what is the difference between that and `"onSave": "test-file"`?
